### DISCUSSION TITLE
Removing the 360 degree limitation

### DISCUSSION THREAD

**Discussion by Mrhansenhahn** [roleoroleo/yi-hack-Allwinner-v2#772] [Q&A]
Hello, is there a way to remove the 360 degree limitation on the movement of the Camera (Yi Dome U) Motors? I would like to use the innards of the camera for another project and use the yi hack for this. Thanks for the help.

**Comment by roleoroleo** [comment]
If you mean endless looping, I think it's not possible.

**Comment by Mrhansenhahn** [comment]
Hello, I would like to use the motors with the rest of the hardware for a small cam vehicle, so that the motors drive the vehicle, but the rotation of the motors is limited so as not to destroy the camera. I would like to remove this limitation so that the vehicle can travel a little further than a few centimeters.

**Comment by roleoroleo** [comment]
Sorry but I don't know.
